homeassistant/components/google/__init__.py

```diff
@@ -79,16 +69,10 @@
 
 EVENT_CALENDAR_ID = "calendar_id"
 
-NOTIFICATION_ID = "google_calendar_notification"
-NOTIFICATION_TITLE = "Google Calendar Setup"
-GROUP_NAME_ALL_CALENDARS = "Google Calendar Sensors"
-
 SERVICE_ADD_EVENT = "add_event"
 
 YAML_DEVICES = f"{DOMAIN}_calendars.yaml"
 
-TOKEN_FILE = f".{DOMAIN}.token"
-
 PLATFORMS = [Platform.CALENDAR]
 
```

Comment:
We can use `cv.removed` instead of `cv.deprecated` in the config schema below.

Reply:
Marked as removed. (Whoops, I didn't leave any of that there intentionally, just forgot after making all the tests pass w/o config).